cmd/sos.go

```diff
@@ -1,40 +1,142 @@
 package cmd
 
 import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
 	"os"
 	"strings"
+	"time"
 
 	minio "github.com/minio/minio-go/v6"
 	"github.com/spf13/cobra"
+	"golang.org/x/net/http2"
 )
 
 const (
-	sosZone = "ch-dk-2"
+	defaultSOSZone = "ch-dk-2"
 )
 
 // sosCmd represents the sos command
 var sosCmd = &cobra.Command{
 	Use:   "sos",
 	Short: "Simple Object Storage management",
+	Long: `Manage Exoscale Object Storage (SOS)
+
+IMPORTANT: Due to a bug in the Microsoft Windows support in the Go
+programming language (https://github.com/golang/go/issues/16736), some
+security certificates such as Exoscale SOS API's one are not recognized as
+being signed by a trusted Certificate Authority. In order to work around this
+issue until the bug is fixed upstream, Windows users are required to use the
+"--certs-file" flag with a path to a local file containing Exoscale SOS API's
+certificate chain to the "exo sos" commands. This file can be downloaded from
+the following address:
+
+	https://www.exoscale.com/sos.crt
```

Comment:
The actual address is up for debate, could be a link to the Community documentation. cc @llambiel @brutasse

Reply:
At some point (not so long in the future), we'll need some PKI-specific folder or hostname to distribute all PKI-related material (be it external, like the case at hand, or internal, if we choose to keep following that way).
(PS: this PKI material may be provided via an http**s**://... URI as long as there is no chicken-'n-egg problem)

Reply:
IIRC the `exoscale.com` cert is universally trusted, only the `*.exo.io` causes issues.